libs/components/forms/src/lib/modules/file-attachment/file-drop.component.ts

```diff
@@ -178,6 +179,23 @@ export class SkyFileDropComponent implements OnInit, OnDestroy {
   @Input({ transform: booleanAttribute })
   public required = false;
 
+  /**
+   * The content of the help popover. When specified along with `labelText`, a [help inline](https://developer.blackbaud.com/skyux/components/help-inline)
+   * button is added to date range picker. The help inline button displays a [popover](https://developer.blackbaud.com/skyux/components/popover)
```

Comment:
should be `file attachment` not `date range picker`

Reply:
fixed